Guide on how to to upgrade herokuish from source is outdated

## Guide on how to to upgrade herokuish from source is outdated

### Steps to Reproduce

Per https://github.com/dokku/dokku/blob/e0c2e7846a551c21fab7595737ae3b988d59578e/docs/getting-started/upgrading.md#upgrading-from-source, following _"To upgrade Herokuish from source, upgrade with"_:

```
root@hz1 /etc/apt # cd /tmp
root@hz1 /tmp # git clone https://github.com/gliderlabs/herokuish.git
Cloning into 'herokuish'...

remote: Enumerating objects: 5445, done.
remote: Counting objects: 100% (636/636), done.
remote: Compressing objects: 100% (386/386), done.
remote: Total 5445 (delta 326), reused 401 (delta 191), pack-reused 4809
Receiving objects: 100% (5445/5445), 1.96 MiB | 13.40 MiB/s, done.
Resolving deltas: 100% (2856/2856), done.
root@hz1 /tmp # cd herokuish
root@hz1 /tmp/herokuish # git pull origin master

From https://github.com/gliderlabs/herokuish
 * branch            master     -> FETCH_HEAD
Already up to date.
root@hz1 /tmp/herokuish # IMAGE_NAME=gliderlabs/herokuish BUILD_TAG=latest VERSION=master make -e build-in-docker
make: *** No rule to make target 'build-in-docker'.  Stop.
```

#### Actual Results

Build fails. (meaning the guide is apparently outdated)

#### Expected Results

Build completes, local herokuish image is updated.

### Additional info

The published herokuish ubuntu package uses buildpack-static v14 which fails to compile (that was fixed in v15, but herokuish with the updated buildpack has not been yet published). That's why I decided to upgrade herokuish locally.

--- a/docs/getting-started/upgrading.md
+++ b/docs/getting-started/upgrading.md
@@ -118,6 +118,5 @@ To upgrade Herokuish from source, upgrade with:
 cd /tmp
 git clone https://github.com/gliderlabs/herokuish.git
 cd herokuish
-git pull origin master
-IMAGE_NAME=gliderlabs/herokuish BUILD_TAG=latest VERSION=master make -e build-in-docker
+CIRCLECI=true IMAGE_NAME=gliderlabs/herokuish BUILD_TAG=latest make build/docker
 ```

